content/v2/certificates.markdown

```diff
@@ -136,3 +136,196 @@ Responds with HTTP 200, renders the certificate private key.
 ~~~json
 <%= pretty_print_fixture("/getCertificatePrivateKey/success.http") %>
 ~~~
+
+## Purchase a Let's Encrypt certificate {#letsencrypt-purchase}
+
+    POST /:account/domains/:domain/certificates/letsencrypt
+
+Purchase a [Let's Encrypt](https://dnsimple.com/letsencrypt) certificate with DNSimple.
+
+<note>
+Please note that the domain must be [delegated](https://support.dnsimple.com/articles/pointing-domain-to-dnsimple/) to DNSimple.
+</note>
+
+The certificate is **free of charge**, but some features of the certificate may require to be subscribed to an higher plan.
+
+### Certificate name
+
+The default certificate name is `www`, which is valid for the naked domain as well.
+The name of the certificate will be `www.example.com`, which is valid both for `https://example.com` and `https://www.example.com`.
+
+You can choose a **custom name** (like `api`), which is valid only for `https://api.example.com`.
+Custom name requires a subscription to Professional or a Business plan.
+
+### Alternate names
+
+A certificate can be purchased for multiple subdomains.
+We call them _alternate names_ or [Subject Alternative Name (SAN)](https://support.dnsimple.com/articles/what-is-ssl-san/).
+
+By default a certificate doesn't have alternate names.
+
+You can purchase a single certificate for both `https://docs.example.com` and `https://status.example.com`, alongside with `https://example.com`.
+Alternate names require a subscription to Professional or a Business plan.
+
+### Auto renewal
+
+By default a certificate isn't auto-renewed when in expires.
+
+If that is your case, you must [manually renew](#letsencrypt-purchase-renewal) the certificate.
+
+Otherwise you can **purchase once** and let the system to renew it automatically.
```

Comment:
I'd expand this sentence to :point_right: _Otherwise you can **purchase the certificate once** and then let the system automatically renew it before it expires. You will still have to install the renewed certificate._

Probably a link to a support doc may come in handy also.

Reply:
@jacegu Which support doc?

Reply:
Not sure, I was wondering if one existed explaining how the process to get a LE certificate works (because the purchase+issue cycle sounds confusing).